src/main/java/Duke.java

```diff
@@ -1,10 +1,210 @@
+import java.util.Scanner;
+import java.util.ArrayList;
+
 public class Duke {
+    // Initialise the task list
```

Comment:
Would it be better if you phrase your comment as a noun eg. `The task list`? Because I think verbs should be reserved for methods.